frontend/src/lib/taxonomy.tsx

```diff
@@ -921,6 +921,136 @@ export const CORE_FILTER_DEFINITIONS_BY_GROUP = {
             ),
             examples: ['01:04:12'],
         },
+        $min_timestamp: {
+            label: 'First timestamp',
+            description: <span>The timestamp of the first event from this session.</span>,
+            examples: [new Date().toISOString()],
+        },
+        $max_timestamp: {
+            label: 'Last timestamp',
+            description: <span>The timestamp of the last event from this session</span>,
+            examples: [new Date().toISOString()],
+        },
+        $entry_url: {
+            label: 'Entry URL',
+            description: <span>The first URL visited in this session</span>,
+            examples: ['https://example.com/interesting-article?parameter=true'],
+        },
+        $exit_url: {
+            label: 'Entry URL',
+            description: <span>The last URL visited in this session</span>,
+            examples: ['https://example.com/interesting-article?parameter=true'],
+        },
+        $initial_utm_source: {
```

Comment:
Actually the `$initial_utm_...` ones shouldn't be needed, because we have automatic handling for these, which is based on the plain `initial_utm_...` definitions

Reply:
Those don't exist under sessions, only events and persons. Sessions don't have the non-$initial ones, they *only* have the initial ones. I could rewrite to use the existing definitions though.

Reply:
I'm really worried about maintaining multiple descriptions of essentially the same property, as historically this has led to all sorts of inconsistencies. Pushed a commit with the kind of abstraction I was thinking of – how does it look to you?

Reply:
I've tweaked it a bit, it's more accurate to pull these in from person properties than session properties, let me know what you think

Reply:
It's now from event properties again